seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -728,6 +730,12 @@ def _test_errors(self, url, fields, workspace_name):
         response_json = response.json()
         self.assertListEqual(response_json['errors'], ['Missing required columns: Affected, Sex'])
 
+        self.mock_load_file.return_value = LOAD_SAMPLE_DATA + MISSING_REQUIRED_SAMPLE_DATA
+        response = self.client.post(url, content_type='application/json', data=json.dumps(REQUEST_BODY))
+        self.assertEqual(response.status_code, 400)
+        response_json = response.json()
+        self.assertListEqual(response_json['errors'], ['Missing Sex in row #4', 'Missing Affected in row #4'])
```

Comment:
It should test error messages like `Invalid value '' for ... in row #n` following the missing error messages if there is no `continue` after the missing column error.

Reply:
those error messages don't exist, we have handing for blank values so they aren't generally invalid. We also wouldn't want there to be more errors here, this is exactly what we would want to tell users if they were missing the values